.gitmodules

```diff
@@ -27,7 +27,7 @@
 	url = https://github.com/amazon-freertos/pkcs11.git
 [submodule "Middleware/AWS/ota"]
 	path = Middleware/AWS/ota
-	url = https://github.com/aws/ota-for-aws-iot-embedded-sdk
+	url = https://github.com/ActoryOu/ota-for-aws-iot-embedded-sdk
```

Comment:
This shouldn't change right?

Reply:
Yeah, that should be changed back to official repo.
I'll do that when official label is created.

Reply:
Fixed.